src/org/jitsi/impl/neomedia/rtp/FrameDesc.java

```diff
@@ -168,9 +192,17 @@ boolean update(RawPacket pkt)
         boolean changed = false;
 
         int seqNum = pkt.getSequenceNumber();
+        byte[] buf = pkt.getBuffer();
+        int off = pkt.getOffset(), len = pkt.getLength();
+
+        MediaStreamImpl stream = rtpEncoding.getMediaStreamTrack()
+            .getMediaStreamTrackReceiver().getStream();
+
+        boolean isSOF = stream.isStartOfFrame(buf, off, len);
         if (minSeen == -1 || RTPUtils.sequenceNumberDiff(minSeen, seqNum) > 0)
         {
             changed = true;
+            inOrder = isSOF;
```

Comment:
If I am reading this correctly inOrder always coincides with start!=-1. Is this on purpose? I don't get the semantics of inOrder.